src/main/java/reposense/util/SystemUtil.java

```diff
@@ -12,6 +15,18 @@ public static boolean isWindows() {
         return (System.getProperty("os.name").toLowerCase().indexOf("win") >= 0);
     }
 
+    /**
+     * Returns true if the given String is a valid URL.
+     */
+    public static boolean isValidUrl(String possibleUrl) {
+        try {
+            new URL(possibleUrl);
+        } catch (MalformedURLException mue) {
+            return false;
+        }
+        return true;
+    }
```

Comment:
Interesting way to determine if URL is valid, was at first hesitant of using try-catch to determine that, but it seems to be a common way to do it.